src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] arrayOfFigure = new Figure[6];
+        final int First_Half_Of_Array = 3;
+
+        for (int i = 0; i < First_Half_Of_Array; i++) {
+            arrayOfFigure[i] = FigureSupplier.getRandomFigure();
```

Comment:
Don't use static methods in your solution. Instead of calling 'FigureSupplier.getRandomFigure()' statically, create an instance of 'FigureSupplier' and call the method on it.